hugo.yaml

```diff
@@ -57,7 +57,7 @@ params:
     collector services to capture distributed traces and metrics from your
     application. You can analyze them using Prometheus, Jaeger, and other
     observability tools.
-  font_awesome_version: 5.8.1
+  font_awesome_version: 6.5.1
```

Comment:
I guess this doesn't break anything, @open-telemetry/docs-maintainers ?

Reply:
Personally I didn't know how to test this change. I didn't see any reference to it in code, and it seems like the docsys dependency is what really drives the FontAwesome icons in the hugo app. But I updated this regardless. I'm not familiar enough with hugo to know if it's needed.

Reply:
You can check using the preview: https://deploy-preview-3649--opentelemetry.netlify.app/

Reply:
Good to know! I did test it out locally when making an update to `/theme/docsys/package.json` locally. And I confirmed that new logo shows up in the footer. I guess now we wait for the upstream [PR 1765](https://github.com/google/docsy/pull/1765) to be merged.

Reply:
@svrnm  what I meant by "I didn't know how to test this change" was I didn't know what specifically the `font_awesome_version` line of code did

Reply:
Ohhh, that line shouldn't be there anymore! I'm glad that this PR allowed us to pinpoint that. See:

- #3653